packages/expo/package.json

```diff
@@ -49,11 +49,11 @@
     "bugsnag-build-reporter": "^1.0.1"
   },
   "devDependencies": {
-    "expo-constants": "~13.2.3"
+    "expo-constants": "~14.0.2"
   },
   "peerDependencies": {
-    "expo": ">=33.0.0",
-    "expo-constants": "~13.2.3",
+    "expo": "~47.0.3",
```

Comment:
I don't think it makes sense to be this restrictive — why would we not allow `47.0.0` or a hypothetical `47.1.0`?

Reply:
I'd like to suggest using `47.x.x` notation also to make it easier to understand what to expect

Reply:
have updated to `^47.0.0` to include all minor and patch versions - and have stuck with this semantic versioning syntax to keep consistency across all of our dependencies